Makefile

```diff
@@ -30,7 +30,7 @@ E2E_PROVIDER_IMAGE_NAME ?= e2e-provider
 # Release version is the current supported release for the driver
 # Update this version when the helm chart is being updated for release
 RELEASE_VERSION := v1.0.1
-IMAGE_VERSION ?= v1.0.1
+IMAGE_VERSION ?= v1.0.1.1
```

Comment:
bumping this image to a patch so we don't build images with the same existing tags

Reply:
I think making it `v1.0.2` for here and in the `docker/Makefile` would maek more sense as i don't think we ever having 4 version number fields.

Reply:
If we make it `v1.0.2`, when we cherry pick to release branch, `v1.0.2` will be published. When we do want to release `v1.0.2` later, how do we trigger the rebuild with the newer changes?

Reply:
ah because it will only rebuild when the file here changes right? hm okay

Reply:
correct! Maybe we can discuss changing the trigger to be more specific (for a file?) I think if we make it trigger on `docker/Makefile`, that'll work too.